requirements/edx/github.txt

```diff
@@ -38,15 +39,15 @@ git+https://github.com/edx/lettuce.git@django1.8/upgrade#egg=lettuce==1.8
 -e git+https://github.com/edx/codejail.git@6b17c33a89bef0ac510926b1d7fea2748b73aadd#egg=codejail
 -e git+https://github.com/edx/js-test-tool.git@v0.1.6#egg=js_test_tool
 -e git+https://github.com/edx/event-tracking.git@django1.8-upgrade#egg=event-tracking
-git+https://github.com/edx/django-splash.git@ammar/upgrade-to-django1.8#egg=django-splash==0.2
+-e git+https://github.com/edx/django-splash.git@ammar/upgrade-to-django1.8#egg=django-splash==0.2
```

Comment:
@macdiesel Master has a `-e` but it was removed when we upgraded `django-splash`. However, this causes migrations to fail – maybe because the image being used for the bokchoy tests already has splash installed in the /src/ folder.